<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>503</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IA</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Verificar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idGeneracion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Num generacion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de Generaciones</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inicioIntervalo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingrese el Intervalo DE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="presicionText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>presicion...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Presicion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="finalIntervalo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingrese el Intervalo HASTA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad en BITS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cantidadText">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="cantidadBitsText">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idPoblacionInicial">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Poblacion Inicial...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Poblacion Inicial</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idPoblacionMaxima">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Poblacion Maxima</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Poblacion Maxima</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_graficar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Graficar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecione Maximo o minimo</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximo</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idPromedio">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Probabilidad de desendencia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PMI</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idmutacionindividuo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Probabilidad de mutacion del individuo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idPMG">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Probabilidad de mutacion de gen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PMG</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
